packages/pyroscope-flamegraph/src/FlameGraph/FlamegraphRenderer.module.scss

```diff
@@ -33,6 +33,48 @@
   display: flex;
 }
 
-.tablePane {
+.tablePane,
+.sandwichPane {
   flex: 1;
 }
+
+.sandwichPane {
+  .sandwichPaneInfo {
+    display: inline-block;
+    position: relative;
+    margin: 30px 0 0 10px;
+    padding: 8px;
+    border-radius: 5px;
+    background-color: var(--ps-sandwich-pane-info-bg);
+    color: var(--ps-neutral-1);
+
+    &::before {
+      content: '◀';
```

Comment:
Nit: I'd maybe use the [border trick](https://css-tricks.com/snippets/css/css-triangle/) instead here simply because this arrow might render differently on different OSs